### ISSUE TITLE
Test failure in Statoil

### ISSUE THREAD

**Issue by joakim-hove** [OPM/opm-material#92]
```
5/12 Test #12: test_immiscibleflash .............   Passed    0.30 sec
 6/12 Test #10: test_components ..................   Passed    0.40 sec
 7/12 Test  #9: test_2dtables ....................***Exception: Other  0.71 sec
terminate called after throwing an instance of 'Opm::NumericalProblem'
  what():  [/private/jenkins/jenkins/RH6/workspace/opm-material-install-RH6/opm/material/common/UniformXTabulated2DFunction.hpp:255] Attempt to get undefined table value (-9.8, -10)
```

**Comment by atgeirr** [comment]
Is this the master or release branch?

**Comment by joakim-hove** [comment]
> Is this the master or release branch?

This is the master branch. Will try the release branch now.

**Comment by joakim-hove** [comment]
Release branch is fine.

**Comment by andlaus** [comment]
does #93 fix it?

**Comment by joakim-hove** [comment]
Yes - #93 fixes it!

**Comment by andlaus** [comment]
ok, merged #93. closing this  issue, too.
